tests/suite/elpi_api.ml

```diff
@@ -3,7 +3,7 @@
 (* ------------------------------------------------------------------------- *)
 open Suite
 
-let declare = Test.declare ~category:(Filename.(chop_extension (basename __FILE__)))
+let declare = Test.declare ~category:(Filename.(chop_extension (basename __FILE__))) ~promote:false
```

Comment:
I home `Test.declare` can have an optional argument `?(promote=false)` of type `?promote:bool` and avoid the long diff